src/validation_workflow/docker/check_docker_daemon.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+if (! docker stats --no-stream 2>/dev/null); then
```

Comment:
We should completely remove this script.
We should just do a simple check to see if docker is online, if not, exit.
There is no reason to keep waiting infinitely and even try to start docker for user.

Reply:
Thanks for the comment, @peterzhuamazon .. I was thinking we could help user to bring up the docker daemon on the background, but sure I could just exit the script if the daemon was not running at the time.